### PR TITLE
Ensure pip installation on build

### PR DESCRIPTION
**Pull request by AliceYeh12**
### What changes are proposed in this pull request, and why are they necessary?
coral-dbt fails on machines that do not have pip installed so we need to add pip installation to the build process.

### How was this patch tested?
The fix resolves the Jenkins build failure.

### CODE REVIEW

File: coral-dbt/build.gradle [hunk 1]
```diff
@@ -12,43 +12,74 @@ spotless {
...
-    '-m',
-    'pip',
-    'install',
-    '-r',
-    'requirements.txt'
-  ])
-  installModulesProcessBuilder.directory(file('src/main/resources/tests'))
-  def installModulesProcess = installModulesProcessBuilder.start()
-  installModulesProcess.waitFor()
+  try {
```
Comment (aastha25) [inline coral-dbt/build.gradle:33]: this block shouldnt be triggered if the previous try-catch failed?

Comment (AliceYeh12) [reply coral-dbt/build.gradle:33]: I was thinking in case it is a permissions problem but the user already has pip installed.

Comment (aastha25) [reply coral-dbt/build.gradle:33]: but we didnt validate if pip is installed.

Comment (AliceYeh12) [reply coral-dbt/build.gradle:33]: Yes, if that is the case, the second try-catch block will catch that.

**Review by aastha25 — APPROVED** [review]
LGTM

**Pull request merged**